apps/dh/e2e-dh/project.json

```diff
@@ -8,7 +8,8 @@
       "options": {
         "playwrightConfig": "apps/dh/e2e-dh/playwright.ts",
         "tsConfig": "apps/dh/e2e-dh/tsconfig.json",
-        "devServerTarget": "app-dh:serve:development"
+        "devServerTarget": "app-dh:serve:development",
+        "exclude": "@b2c"
```

Comment:
suggestion: I was confused by the `@` character because I thought it has a special meaning for grep. From what I understand, it's only used to find the right tests to run. In order to make it more clear, let's add a prefix to the healthcheck test. Something like `[B2C Healthcheck]` and update the `exclude`/`include` values accordingly.